mergify_engine/actions/merge/queue.py

```diff
@@ -188,20 +189,19 @@ def handle_first_pull_in_queue(self, ctxt):
             c for c in ctxt.pull_engine_check_runs if c["name"].endswith(" (merge)")
         ]
 
-        output = helpers.merge_report(ctxt, True)
-        if output:
-            conclusion, title, summary = output
+        result = helpers.merge_report(ctxt, True)
+        if result:
             ctxt.log.info(
                 "pull request closed in the meantime",
-                conclusion=conclusion,
-                title=title,
-                summary=summary,
+                conclusion=result.conclusion,
+                title=result.title,
+                summary=result.summary,
```

Comment:
could do `result=result` though

Reply:
You read in my mind ;)

Reply:
Not really I guess?

Reply:
Oh I change `set_check_run` and `set_summary_check` signature to take `result` directly. But I didn't do it for logging to keep these attributes searcheable in datadog.